.eslintrc.json

```diff
@@ -11,6 +11,8 @@
     "no-console": "off",
     "require-jsdoc": "off",
     "valid-jsdoc": "off",
+    "no-unreachable-loop": "off",
+    "no-promise-executor-return": "off",
```

Comment:
Do these need a closer look? I'll take a look at refactoring the code.

Reply:
Yeah they might, I defaulted these to off and didn't review `new Promise` usaages since the other rules in this file were off.

Reply:
Whoops, thanks!